charts/signoz/templates/otel-collector/deployment.yaml

```
@@ -116,6 +116,9 @@ spec:
           volumeMounts:
             - name: otel-collector-config-vol
               mountPath: /conf
+            {{- if .Values.otelCollector.extraVolumeMounts }}
+            {{ toYaml .Values.otelCollector.extraVolumeMounts | nindent 12 }}
+            {{- end }}
```

minor indentation pattern suggestion

```suggestion
            {{- if .Values.otelCollector.extraVolumeMounts }}
              {{ toYaml .Values.otelCollector.extraVolumeMounts | nindent 12 }}
            {{- end }}
```

---

Done in : https://github.com/SigNoz/charts/pull/270/commits/3379cd38b5cdfd002cb6c8a0097cc309320b0f00

---

Done in: https://github.com/SigNoz/charts/pull/270/commits/4d9fdd7b27779b00b7245554d0aade527db1bffc